### PR TITLE
Edits for QS-PC

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10776

Integrating suggestions from @paususe 
[suse_manager_quickstart-public-cloud_guide-1.pdf](https://github.com/uyuni-project/uyuni-docs/files/4296614/suse_manager_quickstart-public-cloud_guide-1.pdf)

### CODE REVIEW

File: modules/installation/pages/pubcloud-requirements.adoc [hunk 1]
```diff
@@ -41,6 +41,7 @@ Only machines in your specified IP ranges must be able to access the instance.
 
 [WARNING]
 ====
+When you run {productname} on a public clouds, you must apply security measures to limit access to the instance.
```
Comment (keichwa) [inline modules/installation/pages/pubcloud-requirements.adoc]:
```suggestion
When you run {productname} on public clouds, you must apply security measures to limit access to the instance.
```

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 2]
```diff
@@ -28,5 +30,43 @@ When you have logged in to the administrator account for the first time, change
 
 
 Add the required software products to your client, and schedule a repository synchronization.
-You might also need to set up custom repositories and channels to manage {ubuntu} or {rhel} products.
-For more on custom channels, see xref:administration:custom-channels.adoc[].
+
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc]:
```suggestion



```

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 3]
```diff
@@ -6,18 +6,57 @@ Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) m
...
+| Microsoft Azure       |
+|===
+
+Select a public cloud instance type with at least:
+
+* 32{nbsp}GB RAM
+* 8 cores
+
+If you are not using the instance for production, then you will require at least:
+
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/setup.adoc]: There is not any difference 😉 Maybe, for simple tests 16GB and 4 cores are enough.  To be approved by someone else.

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 4]
```diff
@@ -6,18 +6,57 @@ Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) m
...
+* 32{nbsp}GB RAM
+* 8 cores
+
+ For more information about hardware requirements, see xref:installation:hardware-requirements.adoc[].
 
 [WARNING]
 ====
+When you run {productname} on a public clouds, you must apply security measures to limit access to the instance.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/setup.adoc]:
```suggestion
When you run {productname} on public clouds, you must apply security measures to limit access to the instance.
```

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 5]
```diff
@@ -6,18 +6,57 @@ Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) m
...
 ====
 
-To access the {productname} {webui}, allow HTTPS when configuring the network access controls.
 
-We recommend that the repositories and the database for {productname} are stored on a seperate storage device.
-This will help to avoid data loss in cases when the {productname} instance is terminated.
-You must set up the storage device before you run the YaST {productname} setup procedure.
 
-For more on setting up a seperate storage volume, see xref:installation:pubcloud-requirements.adoc[].
+To access the {productname} {webui}, configurr the network access controls to allow HTTPS.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/setup.adoc]:
```suggestion
To access the {productname} {webui}, configure the network access controls to allow HTTPS.
```

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 6]
```diff
@@ -28,5 +30,45 @@ When you have logged in to the administrator account for the first time, change
...
+====
+
+To synchronize, navigate to menu:Software[Manage > Channels], and for each channel you created, click it and:
+
+. Navigate to the [guimenu]``Repositories`` tab.
+. Navigate to the [guimenu]``Sync`` sub-tab, and click btn:[Sync Now] to begin synchronization immediately.
+
+You can also schedule synchronization from this screen.
+
+endif::[]
```
Comment (paususe) [inline modules/quickstart-public-cloud/pages/installation.adoc]: This makes the build fail. Where is the matching `if[]` ?

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 7]
```diff
@@ -6,18 +6,52 @@ Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) m
...
 
-Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords ``suse``, ``manager``, ``proxy``, or ``BYOS``.
-Select a public cloud instance type that meets the hardware requirements in xref:installation:hardware-requirements.adoc[].
+Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords ``suse``, ``manager``, or ``BYOS``.
+
+.SUSE Manager Image Names
+[cols="1,1", options="header"]
+|===
+| Provider              | Image Name
+| Amazon EC2            |
```
Comment (paususe) [inline modules/quickstart-public-cloud/pages/setup.adoc]: Why are the image names empty?

File: modules/quickstart-public-cloud/pages/installation.adoc (hunk 6 above)
Comment (Loquacity) [reply modules/quickstart-public-cloud/pages/installation.adoc]: that was just copypasta

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 8]
```diff
@@ -6,18 +6,52 @@ Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) m
...
-This will help to avoid data loss in cases when the {productname} instance is terminated.
-You must set up the storage device before you run the YaST {productname} setup procedure.
 
-For more on setting up a seperate storage volume, see xref:installation:pubcloud-requirements.adoc[].
+To access the {productname} {webui}, configure the network access controls to allow HTTPS.
+
+Amazon EC2:
+
+----
+TBD
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/quickstart-public-cloud/pages/setup.adoc]: SME required, I don't have the info.

File: modules/quickstart-public-cloud/pages/setup.adoc (hunk 7 above)
Comment (Loquacity) [reply modules/quickstart-public-cloud/pages/setup.adoc]: I don't have the info.

**Comment by Loquacity** [comment]
Further to conversation with Pau, I'm going to merge this and come back to it when I have more bandwidth. The whole thing needs to have the language changed.

**Pull request merged**